proto/ibc/core/channel/v1/query.proto

```diff
@@ -374,3 +380,16 @@ message QueryNextSequenceReceiveResponse {
   // height at which the proof was retrieved
   ibc.core.client.v1.Height proof_height = 3 [(gogoproto.nullable) = false];
 }
+
+// QueryUpgradeSequenceRequest is the request type for the QueryUpgradeSequence RPC method
+message QueryUpgradeSequenceRequest {
+  string port_id    = 1;
+  string channel_id = 2;
+}
+
+// QueryUpgradeSequenceResponse is the response type for the QueryUpgradeSequence RPC method
+message QueryUpgradeSequenceResponse {
+  uint64                    upgrade_sequence = 1;
+  bytes                     proof            = 2;
```

Comment:
do we still need the proof in the response proto?

Reply:
an unrelated question: do we need the cli commands for this query as well?

Reply:
> I believe the reason is that there is not support in SDK/Comet to generate proofs for a given key/value . @colin-axner opened https://github.com/cosmos/cosmos-sdk/issues/7036 in the SDK a while back.

yes, I remember this issue from the relayers having to always query the tendermint rpc instead of being able to query proofs from the grpc endpoint. I guess the question is if we want to plan for a (possible?) future of a proof returned via grpc by still putting in the proof field in the response body, even if now it will always return nil, or just remove that proof field.